lib/db.spec.ts

```diff
@@ -0,0 +1,9 @@
+import db from "./db";
```

Comment:
Atleast to me, renaming default imports unless necessary (namespace clash etc.) seems unneeded, and adds a layer of confusion. Otherwise LGTM ✨

Reply:
Changed it so that the db module exports `pool` which is used as is.